projects/containernetworking/plugins/Makefile

```diff
@@ -22,8 +12,13 @@ binaries:
 tarballs:  binaries
 	build/create_tarballs.sh $(REPO) $(GIT_TAG)
 
+.PHONY: build
+build: tarballs
+	echo "Done $(COMPONENT)"
+
 .PHONY: release
 release: tarballs
+	echo "Done $(COMPONENT)"
```

Comment:
So the release target is not uploading to S3 vs build? What is the situation there?

Reply:
That is the change I put in on #137 which hasn't merged yet

Reply:
to confirm, you are saying once 137 is merged, the release target will upload after build?  Thats how it looks [here](https://github.com/aws/eks-distro/pull/137/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R66)

Reply:
This change is just adding a build target to the Make file so the presubmit tests will pass for #137 The #137 PR adds the upload to s3 like you pointed out in your link